benchmark/run_riatahgt.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# usage: run RIATA-HGT software
+gene_tree_dir=$1
+output_fp=$2
+verbose=$3
+stdout=$4
+stderr=$5
+scripts_dir=$6
+species_tree_fp=$7
+input_file_nex=$8
+output_file=$9
+phylonet_install_dir=${10}
+
+total_user_time_riatahgt="0.0"
+total_wall_time_riatahgt="0.0"
+i=0
+printf "#RIATAHGT\n" >> $output_fp
+
+# search for HGTs in each gene tree
+for gene_tree in $gene_tree_dir/*.nwk
+do
+    gene_tree_file=$(basename $gene_tree)
+    gene_number=$(echo $gene_tree_file | sed 's/[^0-9]*//g')
+    printf "$i\t$gene_number\t" >> $output_fp
+
+    python ${scripts_dir}/reformat_input.py --method 'riata-hgt' \
```

Comment:
The reason that you have to run them like this (prepending python) is because they are not executable in the repo. Perhaps it's worth changing.